news/2025/2025-06-24-state-of-the-ranked-section-survey.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: State of the Ranked section survey
+date: 2025-06-24 12:00:00 +0000
+---
+
+Pollers wanted — help us pave the way forward for the future of the Ranked section!
+
+![](/wiki/shared/news/banners/beatmap-management.jpg)
+
+As the [Ranked](/wiki/Beatmap_ranking_procedure) section has quickly become the most varied — and active! — it has ever been over the last couple years, we would like to take a moment to ask the community for their feelings on the section's current state. Admittedly, this is not a task we can tackle alone. Therefore, we would like to touch some base with the community to evaluate what would be the best way to step forward.
+
+The mapping landscape is constantly evolving, and knowing what most of the community wants at any given time is difficult. Answering these questions will take no more than 5 minutes and will be very helpful to us!
+
+[**If you want to get involved, please click this link to get started!**](https://forms.hivie.tn/form/2025-06-ranked-section-poll-public)
+
+While everyone is encouraged to participate in the above poll, [Beatmap Nominators](/wiki/People/Beatmap_Nominators) will be getting their own private poll. While the questions will not be any different, this will help us compare and contrast two sides of one same coin.
```

Comment:
bit more generic especially if we're doing just one poll and filter afterwards

```suggestion
Much like other surveys we've done in the past, we'll break down the results by area in the community: players, mappers, [Beatmap Nominators](/wiki/People/Beatmap_Nominators), and so on. This will help us compare and contrast between perspectives.
```